src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -19,13 +19,8 @@ public class PushToBlobFeed : MSBuild.Task
         [Required]
         public string AccountKey { get; set; }
 
-        [Required]
         public ITaskItem[] ItemsToPush { get; set; }
 
-        public string IndexDirectory { get; set; }
-
-        public bool PublishFlatContainer { get; set; }
```

Comment:
There are still potential scenarios where we want to publish to a flat directory. What is the plan for those cases?

Reply:
In Sleet all packages live in flatcontainers so no need to make this configurable

Reply:
This is exactly the confusion I was talking about. This flat container is a different concept then then flatcontainer folder Sleet produces. This flatcontainer option was just about putting all files in the root directory with no indexing at all.

Reply:
At least CoreCLR will be broken by not having this "flat" option, so they will need to be fixed once your change flows. I do still think we may need to have this option to publish to a flat container, essentially by going around sleet and publish things to the Assets folder for example. What is your expectation for how we publish just to an azure blob container? Would we use a different task for that?